.changeset/breezy-coats-count.md

```diff
@@ -0,0 +1,5 @@
+---
+"octopus-deploy": major
+---
+
+Bumped octopus server version
```

Comment:
Nit: can we make this more descriptive, something like `Updated installed Octopus Server version to `2024.4`.